MonopolyKata/BoardSlot.cs

```diff
@@ -7,12 +7,19 @@ public class BoardSlot
         public Int32 amount;
         public String name;
         public String color;
```

Comment:
Public member variables should be Pascal case. In most cases, you probably want to use public properties vs members; however this appears to be a data structure so its a toss up.